aspnetcore/mvc/controllers/bind-tryparse/7.0-samples/BindUsingTryParse/BindTryParseMVC/Models/DateRange.cs

```diff
@@ -15,7 +12,7 @@ public DateRange(DateOnly fromDate, DateOnly toDate)
             To = toDate;
         }
 
-        public static bool TryParse(string? value, IFormatProvider? provider,
+        public static bool TryParse(string? value, 
```

Comment:
@fiyazbinhasan  I removed the format provider since it's not used. Actually, when I added
`<PackageReference Include="System.Runtime.Experimental" Version="7.0.0-preview.2.22152.2" />`
I got a compile error with the format provider in the code.